Mage.Server/release/config/config.xml

```diff
@@ -149,6 +149,7 @@
         <deckType name="Constructed - Pioneer" jar="mage-deck-constructed.jar" className="mage.deck.Pioneer"/>
```

Comment:
You can fix that (Constructed - Pioneer) too with missing `-${project.version}`.